lib/groonga/client.rb

```diff
@@ -78,6 +78,7 @@ def initialize(options={})
       if protocol == :gqtp
         @connection = Groonga::Client::Protocol::GQTP.new(options)
       else
+        options[:use_ssl] = true if protocol == :https
```

Comment:
Do you mean we don't need to follow `Net::HTTP`'s API `use_ssl` and should convert `use_tls` of Groonga Client to `use_ssl` of Net::HTTP?